src/analysis/out_parameters.rs

```diff
@@ -113,22 +113,27 @@ pub fn analyze(
 pub fn analyze_imports(env: &Env, parameters: &[Parameter], imports: &mut Imports) {
     for par in parameters {
         if par.direction == ParameterDirection::Out {
-            match *env.library.type_(par.typ) {
-                Type::Bitfield(..) | Type::Enumeration(..) => imports.add("std::mem"),
-                Type::Fundamental(fund)
-                    if fund != Fundamental::Utf8
-                        && fund != Fundamental::OsString
-                        && fund != Fundamental::Filename =>
-                {
-                    imports.add("std::mem")
-                }
-                _ if !par.caller_allocates => match ConversionType::of(env, par.typ) {
-                    ConversionType::Direct | ConversionType::Scalar => (),
-                    _ => imports.add("std::ptr"),
-                },
-                _ => (),
-            }
+            analyze_type_imports(env, par.typ, par.caller_allocates, imports);
+        }
+    }
+}
+
+pub fn analyze_type_imports(env: &Env, typ: TypeId, caller_allocates: bool, imports: &mut Imports) {
```

Comment:
Why a new `pub` function that is not used anywhere else (yet)? :)

Reply:
You're right, it doesn't need to be `pub`, however it needs to be a new function since it's recursive on a single out param when the type is `Alias`.